lib/src/main/kotlin/com/swmansion/starknet/service/http/requests/HttpRequest.kt

```diff
@@ -20,7 +21,7 @@ class HttpRequest<T> private constructor(
     private val service: HttpService,
 ) : Request<T> {
 
-    internal constructor(
+    constructor(
```

Comment:
Why make it public?

Reply:
No need, I changed constructor in both `HttpRequest` and `HttpBatchRequest` to internal.